test/unit/stream_alert_rule_processor/test_handler.py

```diff
@@ -0,0 +1,222 @@
+'''
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import base64
+import logging
+
+from mock import call, Mock, mock_open, patch
+
+from nose.tools import (
+    assert_equal,
+    assert_false,
+    assert_list_equal,
+    assert_true,
+    raises
+)
+
+from stream_alert.rule_processor import LOGGER
+from stream_alert.rule_processor.config import ConfigError
+from stream_alert.rule_processor.handler import StreamAlert
+from stream_alert.rule_processor.handler import load_config
+
+
+from unit.stream_alert_rule_processor.test_helpers import (
+    _get_mock_context,
+    _get_valid_event
+)
+
+
+class TestStreamAlert(object):
+    """Test class for StreamAlert class"""
+
+    def __init__(self):
+        self.__sa_handler = None
```

Comment:
Any particular reason this is a double underscore when other private variables are single underscore? Doesn't really matter either way, just be consistent